test/Microsoft.AspNetCore.Http.Abstractions.Tests/PathStringTests.cs

```diff
@@ -200,7 +198,12 @@ public void StartsWithSegmentsWithRemainder_DoesMatchUsingSpecifiedComparison(st
         [InlineData("encoded", "/http%3a%2f%2f[foo]%3A5000/%", "/http%3a%2f%2f%5Bfoo%5D%3A5000/%25")]
         [InlineData("encoded", "/http%3a%2f%2f[foo]%3A5000/%2", "/http%3a%2f%2f%5Bfoo%5D%3A5000/%252")]
         [InlineData("encoded", "/http%3a%2f%2f[foo]%3A5000/%2F", "/http%3a%2f%2f%5Bfoo%5D%3A5000/%2F")]
-        public void ToUriComponentEscapeCorrectly(string category, string input, string expected)
+        public void ToUriComponentEscapeCorrectly(
+#pragma warning disable xUnit1026 // Theory methods should use all of their parameters
+            string category,
```

Comment:
Troy added it for no apparent reason. Kill it.
https://github.com/aspnet/HttpAbstractions/commit/150b4708f158c0cdcc2d49f2ec72f3e82c6c64e4#diff-1388310260e4da60f42f87fde7d9f702R198